descriptors/ModuleDescriptor-template.json

```diff
@@ -121,6 +121,30 @@
         }
       ]
     },
+    {
+      "id": "acquisition-piece-events",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/audit-data/acquisition/piece/{id}",
+          "permissionsRequired": [
+            "acquisition.piece.events.get"
+          ]
+        },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/audit-data/acquisition/piece/{id}/unique-status",
```

Comment:
basically theretically the same piece status can be changed twice and more times and we need to show this in history.
So propose to rename from unique-status to status-change-history

Reply:
Changed